pkg/liqoctl/install/handler.go

```diff
@@ -0,0 +1,50 @@
+package install
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+const providerFlag = "provider"
+
+// HandleInstallCommand implements the install command. It detects which provider has to be used, generates the chart
+// with provider-specific values. Finally, it performs the installation on the target cluster.
+func HandleInstallCommand(cmd *cobra.Command, args []string) {
+	helmClient, config, err := initHelmClient()
+	if err != nil {
+		fmt.Printf("Unable to create a client to the target cluster: %s", err)
+		return
+	}
+	ctx := context.Background()
```

Comment:
The context should be created in the main, configuring the appropriate exit handlers, and then propagated.

Reply:
Since it requires a bit of refactoring, I propose to solve this in a dedicated PR.